src/identity-providers/add/OpenIdConnectSettings.tsx

```diff
@@ -17,39 +17,49 @@ export const OpenIdConnectSettings = () => {
 
   const adminClient = useAdminClient();
   const { realm } = useRealm();
-  const { setValue, errors, setError } = useFormContext();
+  const { setValue, errors, setError, clearErrors } = useFormContext();
 
   const setupForm = (result: any) => {
     Object.keys(result).map((k) => setValue(`config.${k}`, result[k]));
   };
 
   const fileUpload = async (obj?: object) => {
-    if (obj) {
-      const formData = new FormData();
-      formData.append("providerId", id);
-      formData.append("file", new Blob([JSON.stringify(obj)]));
+    clearErrors("discoveryError");
+    if (!obj) {
+      return;
+    }
+
+    const formData = new FormData();
+    formData.append("providerId", id);
+    formData.append("file", new Blob([JSON.stringify(obj)]));
 
-      try {
-        const response = await fetch(
-          `${getBaseUrl(
-            adminClient
-          )}admin/realms/${realm}/identity-provider/import-config`,
-          {
-            method: "POST",
-            body: formData,
-            headers: {
-              Authorization: `bearer ${await adminClient.getAccessToken()}`,
-            },
-          }
-        );
+    try {
+      const response = await fetch(
+        `${getBaseUrl(
+          adminClient
+        )}admin/realms/${realm}/identity-provider/import-config`,
+        {
+          method: "POST",
+          body: formData,
+          headers: {
+            Authorization: `bearer ${await adminClient.getAccessToken()}`,
```

Comment:
I believe that this should start with a capital letter.
```suggestion
            Authorization: `Bearer ${await adminClient.getAccessToken()}`,
```